docs/01-infrastructure-azure.md

```diff
@@ -236,7 +236,7 @@ Create VM
 azure vm create \
 	--resource-group the-hard-way \
 	--name etcd0 \
-	--vm-size Standard_D4 \
```

Comment:
I think the size of the machine is irrelevant. this is not exactly meant for production. @jmspring  you are right ideally we should run with 2 data disks (etcd should run on premium storage). but again this is not meant for production.